components/Header.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+
```

Comment:
I see that you in this component used **css in js** instead of **styled components.** I guess that's because you wanted to try that out as well. That is totally fine for this project, but going forward try to stick to one way of styling your components.